iroh/src/endpoint/connection.rs

```diff
@@ -241,15 +241,16 @@ fn conn_from_quinn_conn(
     let alpn = alpn_from_quinn_conn(&conn).ok_or_else(|| e!(AuthenticationError::NoAlpn))?;
     let paths_info_watchable = init_paths_info_watcher(&conn, ep);
     let paths_info = paths_info_watchable.watch();
-    // Register this connection with the magicsock.
-    ep.msock
-        .register_connection(remote_id, &conn, paths_info_watchable.clone());
-    Ok(Connection {
+    let conn = Connection {
         remote_id,
         alpn,
         inner: conn,
         paths_info,
-    })
+    };
+    let info = conn.to_info();
+    // Register this connection with the magicsock.
```

Comment:
I think we can remove this comment now, so we look a bit less like an AI :sweat_smile: